quinn-proto/src/connection.rs

```diff
@@ -3243,6 +3260,22 @@ struct SentPacket {
 /// Ensures we can always fit all our ACKs in a single minimum-MTU packet with room to spare
 const MAX_ACK_BLOCKS: usize = 64;
 
+/// Events to be sent to the Connection
+pub enum ConnectionEvent {
+    NewIdentifiers([Option<ConnectionId>; LOC_CID_COUNT]),
+}
+
+/// Events to be sent to the Endpoint
+#[derive(Clone, Debug)]
+pub enum EndpointEvent {
```

Comment:
Should we move what is currently `Io::RetireConnectionId` into this enum?

Reply:
Yup, see d27cd156790cbac8af8da12f693dba12bd8b5564 and the cleanup in b676a9fd863edd8325f26bfdf5ad7beba3acbfd9.